diff --git a/node_modules/@sentry/react-native/dist/js/tracing/span.js b/node_modules/@sentry/react-native/dist/js/tracing/span.js
index df32e49..22d15cf 100644
--- a/node_modules/@sentry/react-native/dist/js/tracing/span.js
+++ b/node_modules/@sentry/react-native/dist/js/tracing/span.js
@@ -15,6 +15,7 @@ export const startIdleNavigationSpan = (startSpanOption, { finalTimeout = defaul
         return undefined;
     }
     const activeSpan = getActiveSpan();
+    clearActiveSpanFromScope(getCurrentScope());
     if (activeSpan && isRootSpan(activeSpan) && isSentryInteractionSpan(activeSpan)) {
         logger.log(`[startIdleNavigationSpan] Canceling ${spanToJSON(activeSpan).op} transaction because of a new navigation root span.`);
         activeSpan.setStatus({ code: SPAN_STATUS_ERROR, message: 'cancelled' });
